boto3_fixtures/sns.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module `sns` implements the setup and tear-down of SNS topics via boto3.
+
+Example:
+    >>> import boto3_fixtures as b3f
+    >>> # ... setup `aws` fixture
+    >>>
+    >>> TOPICS = [
+    >>>     "topic-1",
+    >>>     {
+    >>>         "TopicName": "topic-with-attrs",
+    >>>         "Attributes": {"DisplayName": "Topic With Attributes"},
+    >>>     }
+    >>> ]
+    >>> sns = b3f.contrib.pytest.service_fixture(sns, topics=TOPICS)
+"""
+
+import typing as T
```

Comment:
Interesting! I've never seen this done before. Where did you pick this up? Is it a common practice?

Reply:
`T` is by my own preference; this way it's easier to lean on your IDE's autocomplete for remembering annotation names. I don't think it's common practice :shrug: